zkdb/src/storage-engine/ipfs.ts

```diff
@@ -101,10 +102,11 @@ export const newLibP2p = async (
   const config: Libp2pOptions = {
     services: {
       dht: kadDHT({ allowQueryWithZeroPeers: true }),
+      identify: identifyService(),
     },
     start: true,
     addresses: {
-      listen: ['/ip4/127.0.0.1/tcp/0'],
+      listen: ['/ip4/0.0.0.0/tcp/0'],
```

Comment:
Do listen on `0.0.0.0` is dangerous?

Reply:
As far as I know, it is necessary as we want our node to be accessible to the public network

Reply:
Public network can access our node via NAT, so i don't think listen on `0.0.0.0` is a good thing.

Reply:
Maybe I have some gaps in the computer network, but in this case 127.0.0.1 will be enough? Should not we specify the private network address?